blog/2020-01-everything-github/README.md

```diff
@@ -0,0 +1,720 @@
+---
+title: "Everything GitHub: Continues Integration, Deployment and Hosting for your Angular App"
+author: Johannes Hoppe
+mail: johannes.hoppe@haushoppe-its.de
+bio: "Johannes is one of the two founders of the Angular.Schule and the main developer of angular-cli-ghpages. Recently he has been recognized as a Google Developer Expert (GDE)."
+author2: Dharmen Shah
+mail2: shhdharmen@gmail.com
+twitter2: shhdharmen
+bio2: "Dharmen is a Husband, Techie, Full Stack JavaScript Developer, Foodie, Hindi Pop Music from Ahmedabad in India. In his free-time he participates on the development of angular-cli-ghpages."
```

Comment:
Let me know how much white space is still there. I will think of additional content if needed.
```suggestion
bio2: "Dharmen is a Full Stack JavaScript Developer. His core skills are JavaScript and CSS. He has worked on Angular and React, both.
He has contributed to angular-cli-ghpages docs and likes to contribute to open-source projects.
He is a foodie and loves listening to music, especially Hindi Pop music. In his free time, he likes to cook. He is from Ahmedabad, India."
```